spec/api-auto-updater-spec.js

```diff
@@ -20,18 +20,18 @@ describe('autoUpdater module', function () {
         return done()
       }
 
-      ipcRenderer.once('auto-updater-error', function (event, message) {
-        assert.equal(message, 'Update URL is not set')
+      ipcRenderer.once('auto-updater-error', (event, message) => {
+        expect(message).to.equal('Update URL is not set')
         done()
       })
       autoUpdater.setFeedURL('')
       autoUpdater.checkForUpdates()
     })
   })
 
-  describe('getFeedURL', function () {
-    it('returns a falsey value by default', function () {
-      assert.ok(!autoUpdater.getFeedURL())
+  describe('getFeedURL', () => {
+    it('returns a falsey value by default', () => {
+      expect(!autoUpdater.getFeedURL()).to.equal(true)
```

Comment:
I think that using `expect(value).to.equal(false)` instead of `expect(!value).to.equal(true)` can make code clearer.